### PR TITLE
setup controllers

### PR DESCRIPTION
**Pull request by chudisoft**
[Project requirements](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md#project-requirements)

    Implement the design from the sneak peek wireframes.
        Styling is nice to have but focus on making sure everything functions first. (Note: Ignore login logic, you will be using Devise in the coming milestone)

Note: Analyze the wireframes deeply and try to identify all the necessary elements and how a user would navigate from one page to another

blog_root_users
blog_user_page
blog_user_all_posts
blog_single_post

Use methods that you have created in the [Blog App - processing data in models](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Processing_data_model_project.md).
    Hint: In order to see some posts or comments, you might need to add some data to your database as described [here](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Processing_data_model_project.md#use-models-to-insert-data).

### CODE REVIEW

File: app/views/posts/index.html.erb [hunk 1]
```diff
@@ -1,2 +1,45 @@
...
+                <div class='col-md-12 pt-2'>
+                    <label class='me-2 text-end'>
+                        Comments: <span><%= post.comments_counter %></span>
+                    </label>
+                    <label class='text-end'>
+                        Likes: <span><%= post.likes_counter %></span>
+                    </label>
+                </div>
+                <div class='col-md-12 pt-2'>
+                    <label class=''>
```
Comment (VICTOR-CHUKWU) [inline app/views/posts/index.html.erb:36]: - [ ] your post index page does not look like the given wireframe, kindly fix this views
your design :point_down: 
Screenshot from 2023-11-02 19-56-40

given design :point_down: 
Screenshot from 2023-05-11 17-37-59

File: app/views/posts/show.html.erb [hunk 2]
```diff
@@ -1,2 +1,3 @@
-<p>Here is the details of post</p>
-<!-- Your post details HTML goes here -->
+<p>Details of the post '<%= @post.title %>'</p>
+<p>Title: <%= @post.title %></p>
+<p>Text: <%= @post.text %></p>
```
Comment (VICTOR-CHUKWU) [inline app/views/posts/show.html.erb:3]: - [ ] your post show does not look like give design, kindly fix the view
your design :point_down: 
Screenshot from 2023-11-02 20-01-22
given design :point_down: 
Screenshot from 2023-09-01 21-07-50

**Review by VICTOR-CHUKWU — CHANGES_REQUESTED** [review]
Hi @chudisoft  👋  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- [ ] some views look similar to design

## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Felix45 — APPROVED** [review]
# Approved :green_circle: 
Hi @VICTOR-CHUKWU, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: Congratulations! 🎉 



## Highlights :heavy_check_mark: 
- [x] Application views setup correctly :white_check_mark: 
- [x] Application functions as expected :white_check_mark:  
- [x] Git flow is used  correctly :white_check_mark: 
- [x] Implemented all the required changes from the previous reviewer :white_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**